network/load-balancer/reference-content/configuring-frontends.mdx

```diff
@@ -0,0 +1,94 @@
+---
+meta:
+  title: Configuring your Load Balancer's frontend
+  description: Learn how to choose different settings and configurations for your Load Balancer's frontend depending on your needs
+content:
+  h1: Configuring your Load Balancer's frontend
+  paragraph: Learn how to choose different settings and configurations for your Load Balancer's frontend depending on your needs
+tags: load-balancer
+categories: 
+  - load-balancer
+  - network
+dates:
+  validation: 2023-06-05
+  posted: 2023-06-05
+---
+
+## Load Balancer overview
```

Comment:
@RoRoJ Do you still need this title since this page is focusing on frontend? In staging I see this title but there's no content associated.

Reply:
Removed, and in the other places